metadata/src/main/java/org/apache/kafka/controller/FeatureControlManager.java

```diff
@@ -94,35 +138,6 @@ ControllerResult<Map<String, ApiError>> updateFeatures(
         }
     }
 
-    ControllerResult<Map<String, ApiError>> initializeMetadataVersion(short initVersion) {
```

Comment:
Yea, this was leftover from a previous iteration on the bootstrapping logic where the controller was just getting the initial metadata version.